test/unittests/CMakeLists.txt

```diff
@@ -51,6 +51,8 @@ find_path(GMOCK_INCLUDE_DIR gmock/gmock.h
           PATHS /usr/includes
           PATH_SUFFIXES gmock)
 
+set (CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -Wno-conversion-null")
+
```

Comment:
I've just added this here in order not to lose this checking in the regular code